configure.ac

```diff
@@ -36,7 +36,7 @@ LT_INIT
 dnl Checks for libraries.
 
 dnl Checks for header files.
-AC_CHECK_HEADERS(crypt.h sys/time.h utmp.h \
+AC_CHECK_HEADERS(crypt.h utmp.h \
```

Comment:
`sys/time.h` is included conditionally in [ulimit.c](https://github.com/shadow-maint/shadow/blob/master/libmisc/ulimit.c#L24). I guess nowadays it's safe to get it out of there and include it unconditionally. What do you think?

Reply:
I covered that condition in a subsequent commit that removed the `HAVE_SYS_RESOURCE_H` macro.

Reply:
I saw it while reviewing this commit but I forgot to delete the comment.